src/helpers/store.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunkMiddleware from 'redux-thunk';
+import { createLogger } from 'redux-logger';
+import rootReducer from '../reducers';
+
+const loggerMiddleware = createLogger();
+
+// const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
Don't leave commented code.
Either add the redux dev tools or remove this line.
You can always go to the documentation and get this line again.